hive/src/main/java/com/netease/arctic/hive/op/ReplaceHivePartitions.java

```diff
@@ -133,20 +134,25 @@ public void commit() {
     }
   }
 
-  private void setHiveLocations() {
+  private void setPartitionProperties() {
```

Comment:
How about rename `setPartitionProperties` to `commitPartitionProperties`?

Reply:
yes, i would take it